src/main/target/common_pre.h

```diff
@@ -397,9 +397,7 @@ extern uint8_t _dmaram_end__;
 #endif
 
 #define USE_CMS
```

Comment:
Should we move this also (line 481) as it should be dependent on OSD availability?